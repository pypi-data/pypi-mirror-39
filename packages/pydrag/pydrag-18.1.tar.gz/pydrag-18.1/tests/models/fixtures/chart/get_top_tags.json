{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&limit=10&method=chart.getTopTags&page=2"
            },
            "response": {
                "body": {
                    "string": "{\"tags\":{\"tag\":[{\"name\":\"rock\",\"url\":\"https://www.last.fm/tag/rock\",\"reach\":\"393374\",\"taggings\":\"3934957\",\"streamable\":\"1\",\"wiki\":{}},{\"name\":\"electronic\",\"url\":\"https://www.last.fm/tag/electronic\",\"reach\":\"251646\",\"taggings\":\"2335574\",\"streamable\":\"1\",\"wiki\":{}},{\"name\":\"seen live\",\"url\":\"https://www.last.fm/tag/seen+live\",\"reach\":\"81624\",\"taggings\":\"2128124\",\"streamable\":\"1\",\"wiki\":{}},{\"name\":\"alternative\",\"url\":\"https://www.last.fm/tag/alternative\",\"reach\":\"260201\",\"taggings\":\"2068697\",\"streamable\":\"1\",\"wiki\":{}},{\"name\":\"indie\",\"url\":\"https://www.last.fm/tag/indie\",\"reach\":\"251353\",\"taggings\":\"1995086\",\"streamable\":\"1\",\"wiki\":{}},{\"name\":\"pop\",\"url\":\"https://www.last.fm/tag/pop\",\"reach\":\"222857\",\"taggings\":\"1931373\",\"streamable\":\"1\",\"wiki\":{}},{\"name\":\"female vocalists\",\"url\":\"https://www.last.fm/tag/female+vocalists\",\"reach\":\"167306\",\"taggings\":\"1586278\",\"streamable\":\"1\",\"wiki\":{}},{\"name\":\"metal\",\"url\":\"https://www.last.fm/tag/metal\",\"reach\":\"154710\",\"taggings\":\"1247954\",\"streamable\":\"1\",\"wiki\":{}},{\"name\":\"alternative rock\",\"url\":\"https://www.last.fm/tag/alternative+rock\",\"reach\":\"165949\",\"taggings\":\"1142533\",\"streamable\":\"1\",\"wiki\":{}},{\"name\":\"classic rock\",\"url\":\"https://www.last.fm/tag/classic+rock\",\"reach\":\"136456\",\"taggings\":\"1137203\",\"streamable\":\"1\",\"wiki\":{}}],\"@attr\":{\"page\":\"2\",\"perPage\":\"10\",\"totalPages\":\"269368\",\"total\":\"2693674\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "1375"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:27:46 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
