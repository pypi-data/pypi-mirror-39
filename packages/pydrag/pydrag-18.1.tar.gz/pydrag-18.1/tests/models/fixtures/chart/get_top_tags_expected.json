{
    "data": [
        {
            "name": "rock",
            "reach": 393374,
            "taggings": 3934957,
            "url": "https://www.last.fm/tag/rock",
            "wiki": {}
        },
        {
            "name": "electronic",
            "reach": 251646,
            "taggings": 2335574,
            "url": "https://www.last.fm/tag/electronic",
            "wiki": {}
        },
        {
            "name": "seen live",
            "reach": 81624,
            "taggings": 2128124,
            "url": "https://www.last.fm/tag/seen+live",
            "wiki": {}
        },
        {
            "name": "alternative",
            "reach": 260201,
            "taggings": 2068697,
            "url": "https://www.last.fm/tag/alternative",
            "wiki": {}
        },
        {
            "name": "indie",
            "reach": 251353,
            "taggings": 1995086,
            "url": "https://www.last.fm/tag/indie",
            "wiki": {}
        },
        {
            "name": "pop",
            "reach": 222857,
            "taggings": 1931373,
            "url": "https://www.last.fm/tag/pop",
            "wiki": {}
        },
        {
            "name": "female vocalists",
            "reach": 167306,
            "taggings": 1586278,
            "url": "https://www.last.fm/tag/female+vocalists",
            "wiki": {}
        },
        {
            "name": "metal",
            "reach": 154710,
            "taggings": 1247954,
            "url": "https://www.last.fm/tag/metal",
            "wiki": {}
        },
        {
            "name": "alternative rock",
            "reach": 165949,
            "taggings": 1142533,
            "url": "https://www.last.fm/tag/alternative+rock",
            "wiki": {}
        },
        {
            "name": "classic rock",
            "reach": 136456,
            "taggings": 1137203,
            "url": "https://www.last.fm/tag/classic+rock",
            "wiki": {}
        }
    ],
    "limit": 10,
    "page": 2,
    "total": 2693674
}
